src/GraphRequestUtil.ts

```diff
@@ -41,7 +41,7 @@ export const urlJoin = (urlSegments: string[]): string => {
  */
 
 export const serializeContent = (content: any): any => {
-	const className: string = content === undefined ? undefined : content.constructor.name;
+	const className: string = content === undefined || content === null ? undefined : content.constructor.name;
```

Comment:
```suggestion
	const className = content?.constructor?.name;
```

Reply:
I tried using this but the build failed. Will look at it in a separate PR.

Reply:
interesting, please share your findings when you get a chance. It could be that the TypeScript version we are using does not support this feature.

Reply:
Yeah, I suspect the same reason for the failure. I will let you know the reason.